content/vendor/cli/app create.md

```diff
@@ -0,0 +1,26 @@
+---
+date: 2021-09-10
+linktitle: "app create"
+title: "app create"
+weight: 90105
+draft: false
+---
+
+Create a kots app.
+
+### Usage
+```bash
+replicated app create NAME
+```
+
+| Flag                 | Type | Description |
+|:----------------------|------|-------------|
+| `-h, --help`   |  |          help for the command |
+| `--token string` | |  The API token to use to access your app in the Vendor API (default uses `$REPLICATED_API_TOKEN` env variable) |
```

Comment:
to use to > used

Reply:
to be clear, this was duplicated from what some existing docs said
I'll change it for all of them